coalesce/config/common.py

```diff
@@ -52,6 +52,7 @@ class Common(Configuration):
 
     # Postgres
     DATABASES = {
+        # Can be overridden using DATABASE_URL
         'default': dj_database_url.config(
             default='postgres://postgres:postgres@postgres:5432/postgres',
```

Comment:
ahh I think we need to update this line to pull from the `DATABASE_URL` environment variable

Reply:
If you look at my commit message:

> Instead we use the `dj_database_url` module, which takes a default url
but can be overridden by providing an environment variable called
`DATABASE_URL`. 

It's built in to `dj_database_url` but it's new to me (and possibly a lot of other people) so I added a comment to tell people about `DATABASE_URL`.

Reply:
> This simple Django utility allows you to utilize the 12factor inspired `DATABASE_URL` environment variable to configure your Django application.

From https://pypi.org/project/dj-database-url/

Reply:
Ahhh **now** I understand! Forgive my slowness and thank you for bearing with me :)